addon/themes/default.js

```diff
@@ -54,6 +58,12 @@ export default EmberObject.extend({
     'cell-content': ''
   },
 
+  labels: {
```

Comment:
Can we call this `messages` (I have a plan to move `messages` from `models-table` to `theme`)?

Reply:
Would you want the word Label added to the elements to better call out the purpose
```
  messages: {
    editRowButtonLabel: 'Edit',
    saveRowButtonLabel: 'Save',
    cancelRowButtonLabel: 'Cancel'
  },

```

Reply:
Yes, this would be great.